server.js

```diff
@@ -2966,23 +2995,26 @@ cache(function(data, match, sendBadge, request) {
 }));
 
 // GitHub issues integration.
-camp.route(/^\/github\/issues(-raw)?\/([^\/]+)\/([^\/]+)\/?([^\/]+)?\.(svg|png|gif|jpg|json)$/,
+camp.route(/^\/github\/issues(-pr)?(-closed)?(-raw)?\/([^\/]+)\/([^\/]+)\/?([^\/]+)?\.(svg|png|gif|jpg|json)$/,
 cache(function(data, match, sendBadge, request) {
-  var isRaw = !!match[1];
-  var user = match[2];  // eg, badges
-  var repo = match[3];  // eg, shields
-  var ghLabel = match[4];  // eg, website
-  var format = match[5];
-  var apiUrl = 'https://api.github.com/repos/' + user + '/' + repo;
+  var isPR = match[1];
+  var isClosed = match[2];
+  var isRaw = !!match[3];
+  var user = match[4];  // eg, badges
+  var repo = match[5];  // eg, shields
+  var ghLabel = match[6];  // eg, website
+  var format = match[7];
+  var apiUrl = 'https://api.github.com/' + (isPR ? 'search/issues?q=is:pr+is:' + (isClosed ? 'closed' : 'open') + '+' : 'repos/') + user + '/' + repo;
```

Comment:
Does this mean `/github/issues-closed/badges/shields.svg` ? I don't have a problem here.